src/main/java/cz/cvut/kbss/termit/exception/TermItException.java

```diff
@@ -24,18 +21,37 @@
  */
 public class TermItException extends RuntimeException {
```

Comment:
I would solve it by having a dedicated marker subclass/interface for not logged exception. It would be less flexible, but would leave the code intact. But this is OK as well.

Reply:
One thing - having the same exception logged at some point and not logged at another point seems a bit confusing to me. Maybe the exception class/interface would avoid this potential misuse.